src/test/java/jenkins/plugins/slack/webhook/WebhookEndpointTest.java

```diff
@@ -105,13 +105,14 @@ public void testNoTextPostData() throws Exception {
 
     @Test
     public void testNoTriggerWordPostData() throws Exception {
+        // No trigger word is present, which is the case when Slack "slash commands" are used
```

Comment:
can you add a test for this functionality please?
it appears you've reworked the existing test but changed its behaviour

Reply:
ok, are you still interested in seeing this merged?

Reply:
I'm currently not using the plugin, so I have no stake in getting it merged. But judging from the comment from @logicfox , others may be interested. Supporting "slash commands" seems like an obvious feature to have.